src/Examples/Advanced/CustomUnitModels/methanol_param_VLE.py

```diff
@@ -119,43 +114,37 @@ def build(self):
                                      ('H2', 'C'): 3.19,
                                      ('CH3OH', 'A'): 18.5875,
                                      ('CH3OH', 'B'): 3626.55,
-                                     ('CH3OH', 'C'): -34.29}
+                                     ('CH3OH', 'C'): -34.29} # these Antoine coefficients assume mmHG and K
 
         Cp = self.config.Cp
-        Cv = Cp - self.gas_constant.value
+        Cv = Cp - Constants.gas_constant/1000/pyunits.mol*pyunits.kmol # convert gas constant from J/mol-K to MJ/kmol-K
         gamma = Cp / Cv
 
         self.gamma = Param(within=NonNegativeReals, mutable=True, default=gamma, doc='Ratio of Cp to Cv')
 
-        self.Cp = Param(within=NonNegativeReals, mutable=True, default=Cp, doc='Constant pressure heat capacity [MJ/(kgmol K)]')
+        self.Cp = Param(within=NonNegativeReals, mutable=True, default=Cp,
```

Comment:
Check the PEP8 standard formatting here - I think that if you need to break arguments over multiple lines then the standard is to put each argument on a new line.